src/hpr_routing.erl

```diff
@@ -2,7 +2,16 @@
 
 -export([
     init/0,
-    handle_packet/1
+    handle_packet/1,
+    routing_info_type/1
+]).
+
+-type routing_info() ::
```

Comment:
It's in a little bit of a different shape, but this exists in `hpr_packet_up:packet_type/1`. 
We can use that and not need to pass `RoutingInformation` all over the place.

Reply:
I will look into that.

Reply:
Removed the passing of RoutingInformation.